controller/class.tapestry-controller.php

```diff
@@ -211,7 +235,36 @@ public function updateTapestrySettings($settings)
         $this->_updatePost($tapestry, 'tapestry');
 
         update_post_meta($this->postId, 'tapestry', $tapestry);
-        return $tapestry->settings;
+
+        return $settings;
+    }
+
+    /**
+     * Add A Tapestry Link
+     * 
+     * @param   Object  $link   Tapestry link
+     * 
+     * @return  Object  $link 
+     */
+    public function addTapestryLink($link)
```

Comment:
Could you please move this function before the `updateTapestrySettings` function so it's a bit more organized? All the add functions should be beside eachother as much as possible.